transit_odp/publish/templates/publish/seasonal_services/index.html

```diff
@@ -71,25 +71,22 @@ <h1 class="govuk-heading-xl app-masthead__title">{% trans 'Seasonal service oper
             <div class="govuk-width-container">
                 <div class="govuk-grid-column-full custom_govuk_table_border">
                     {% if seasonal_services_counter %}
-                        {% render_table table %}
+                        <form method="post" enctype="multipart/form-data" action="{% url 'delete-seasonal-service' pk1=pk1 host hosts.publish %}?page={{ page_obj.number }}&">
+                            {% csrf_token %}
+                            {% render_table table %}
+                        </form>
                     {% else %}
-                        <div class="govuk-grid-column-full">
-                            <p class="govuk-body-m">
-                                {% trans "No seasonal services added." %}
-                                </br>
-                                {% trans "Click 'Add new date range' to add a service." %}
-                            </p>
-                        </div>
+                        {% include 'publish/seasonal_services/empty_table_header.html' %}
                     {% endif %}
                 </div>
             </div>
             <div class="govuk-width-container">
                 <div class="govuk-grid-row">
                     <div class="govuk-grid-column-two-thirds">
-                        <a role="button" class="govuk-button govuk-button--secondary govuk-!-margin-top-5" href={% url 'add-seasonal-service' pk1=pk1 host hosts.publish %}>
+                        <a role="button" class="govuk-button govuk-button--secondary govuk-!-margin-top-5" href="{% url 'add-seasonal-service' pk1=pk1 host hosts.publish %}?page={{ page_obj.number }}&">
```

Comment:
I think with the page number code dropped for everything except delete. This can be removed